README.md

```diff
@@ -3,7 +3,7 @@
 [![Maintainability](https://api.codeclimate.com/v1/badges/7d12c4fd472569490788/maintainability)](https://codeclimate.com/github/Drone3D-Team/Drone3D/maintainability)
 [![Test Coverage](https://api.codeclimate.com/v1/badges/7d12c4fd472569490788/test_coverage)](https://codeclimate.com/github/Drone3D-Team/Drone3D/test_coverage)
 
-<img src="https://user-images.githubusercontent.com/44334351/119946869-5f040d00-bf97-11eb-87c7-3761072ddced.jpg" width="50"> 
+<img src="./images/logo.jpg" width="50"> 
```

Comment:
I think the "./" is not necessary here, just putting images/logo.jpg might be clearer

Reply:
You're right, I've just modified it